docs/CMakeLists.txt

```diff
@@ -50,20 +48,17 @@ set(DOXYGEN_TOC_INCLUDE_HEADINGS 4)
 set(DOXYGEN_TYPEDEF_HIDES_STRUCT YES)
 set(DOXYGEN_UML_LIMIT_NUM_FIELDS 20)
 set(DOXYGEN_UML_LOOK YES)
-set(DOXYGEN_WARN_NO_PARAMDOC YES)
-set(DOXYGEN_WARN_IF_UNDOCUMENTED YES)
+set(DOXYGEN_WARN_IF_UNDOCUMENTED NO)
 set(DOXYGEN_WARN_IF_DOC_ERROR YES)
-set(DOXYGEN_WARN_AS_ERROR NO)
+set(DOXYGEN_WARN_IF_INCOMPLETE_DOC YES)
```

Comment:
What is the difference between `DOXYGEN_WARN_IF_UNDOCUMENTED` and `DOXYGEN_WARN_IF_INCOMPLETE_DOC` ? I googled but `DOXYGEN_WARN_IF_INCOMPLETE_DOC` only gives 2 results.

Reply:
Here are the doxygen definitions, the prefix `DOXYGEN` in the option name comes from cmake,
```
WARN_IF_UNDOCUMENTED

    If the WARN_IF_UNDOCUMENTED tag is set to YES then doxygen will generate warnings for undocumented members. If [EXTRACT_ALL](https://www.doxygen.nl/manual/config.html#cfg_extract_all) is set to YES then this flag will automatically be disabled.

    The default value is: YES.

WARN_IF_INCOMPLETE_DOC

    If WARN_IF_INCOMPLETE_DOC is set to YES, doxygen will warn about incomplete function parameter documentation. If set to NO, doxygen will accept that some parameters have no documentation without warning.

    The default value is: YES.```